filesystem/fs_interface.go

```diff
@@ -0,0 +1,14 @@
+package filesystem
+
+type Client interface {
+	Open() File
```

Comment:
we can skip create, can we?

Xiaoyun

On Sun, Mar 8, 2015 at 6:59 AM, Hongchao Deng <notifications@github.com>
wrote:

> In filesystem/fs_interface.go
> <https://github.com/taskgraph/taskgraph/pull/18#discussion_r26000408>:
>
> > @@ -0,0 +1,14 @@
> > +package filesystem
> > +
> > +type Client interface {
> > +	Open() File
>
> FYI, os.Create() returns a File.
> http://golang.org/pkg/os/#Create
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/taskgraph/taskgraph/pull/18/files#r26000408>.
>